src/NuGet.Services.KeyVault/CachingSecretReader.cs

```diff
@@ -44,8 +57,16 @@ public async Task<ISecret> GetSecretObjectAsync(string secretName)
             {
                 return result.Secret;
             }
+
+            var start = DateTimeOffset.Now;
```

Comment:
Shouldn't this always be UTC time?

```suggestion
            var start = DateTimeOffset.UtcNow;
```